clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AnnotationsBasedAtomicOperationsRegistry.groovy

```diff
@@ -37,6 +37,82 @@ class AnnotationsBasedAtomicOperationsRegistry extends ApplicationContextAtomicO
   List<CloudProvider> cloudProviders
 
   @Override
+  AtomicOperationConverter getAtomicOperationConverter(String description, String cloudProvider) {
```

Comment:
This looks like a huge diff but I actually just duplicated the existing body of the now-deprecated `getAtomicOperationConverter` minus the `providerVersion` logic since I'm going to delete the deprecated version next Clouddriver release. Didn't seem worth optimizing for minimal duplicated code for one release.